components/profile-about-tab/profile-about-tab.jsx

```diff
@@ -12,6 +12,7 @@ class ProfileAboutTab extends React.Component {
 
     return this.props.longBio
       .split(`\n`)
+      .filter(s => !!s)
```

Comment:
not related to meta but console was complaining about array item having the same key (empty string) so I decided to fix this